oops/oops_test.go

```diff
@@ -70,6 +71,23 @@ func doubleWrapf() error {
 	return oops.Wrapf(oops.Wrapf(oops.Wrapf(rootCause, "yuck"), "bad"), "why would you do this")
 }
 
+func stripPathPrefix(s string) string {
+	// Strip path prefixes from filenames, aids comparisons in stack trace tests
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("Cant work out where we are")
```

Comment:
But maybe it's a bit more Go idiomatic altogether to return an error:

```suggestion
		return "", errors.New("Cannot determine caller filename")
```

Then, when calling `stripPathPrefix`, the test can assert that it encounters no error.

Alternatively, if you prefer `panic`-ing, I believe there's a typo worth correcting...
```suggestion
		panic("Can't work out where we are")
```